requirements.txt

```diff
@@ -19,13 +19,22 @@ click-spinner>=0.1.8
 click-repl>=0.1.6
 asciitree>=0.3.3
 tabulate>=0.8.2
-prompt-toolkit>=2.0.1; python_version >= '3.4'
-# See pywbemtools issue # 192. The repl mode fails with unexpected Exception
+
+# prompt-toolkit>=2.0 failed on py27 (issue #192), so it was pinned to <2.0.
+#   Later, the fix for issue #224 allowed to lift that pinning.
+# prompt-toolkit>=3.0 does not support py27.
+# prompt-toolkit>=3.0 may cause WinError 995 on py38 on Windows (issue #690).
 prompt-toolkit>=1.0.15,<3.0.0; python_version == '2.7'
+prompt-toolkit>=2.0.1; python_version >= '3.4' and python_version < '3.8'
```

Comment:
Was there a specific reason we set the minimum to 2.0.1 for python 3 and now still keep 1.0.15 for python 2.7?   There was an issue with click_repl (click_repl issue # 34 early in the release of the 2.0 toolkit but that was fixed). Which I think is whyu we specify the minimum as 2.0.1.

Reply:
First, the minimum versions were not changed by this PR, it only added the pinning to <3.0.0 for py38 on Windows.

The reason the minimum versions are different is that at some point, on py27 it was pinned to <2.0.0 so the minimum version naturally stayed at a 1.x version. Later, the pinning could be lifted to <3.0.0 but there was no real reason to increase the minimum version. That's all on py27.

The minimum version for py3 needs to be a 2.x version because that is required to get py3 support.